cypress/locators/index.js

```diff
@@ -32,6 +32,10 @@ export const getComponent = (component) =>
 export const getElement = (element) =>
   cy.get(`[data-element="${element}"]`).first();
 export const closeIconButton = () => cy.get(CLOSE_ICON_BUTTON);
+export const openNestedDialogButton = () =>
```

Comment:
we don't need those functions, rather just create and use `getDataElementByValue("main-text").contains(text)`
like:
```suggestion
export const openDialogByName = (name) => getDataElementByValue("main-text").contains(name)
```